code/modules/mob/living/carbon/xenomorph/Evolution.dm

```diff
@@ -139,12 +139,21 @@
 	else
 		new_xeno.plasma_stored = new_xeno.plasma_max*(plasma_stored/plasma_max) //preserve the ratio of plasma
 
+	for(var/i in src.built_structures)
+		new_xeno.built_structures += built_structures[i]
+
+	new_xeno.built_structures = built_structures.Copy()
+
+	for(var/i in built_structures)
+		var/list/L = built_structures[i]
+		QDEL_NULL_LIST(L)
```

Comment:
are you sure that this is right, because in theory this deletes the spikes. did you test it?